packages/ui-core/src/components/Search/Search.tsx

```diff
@@ -0,0 +1,172 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { useCallback, useState, useRef, useEffect } from 'react';
+import SearchIcon from 'icons/Basic/16/Search_16.svg';
+import debounce from 'lodash.debounce';
+import cnCreate from 'utils/cnCreate';
+import './Search.less';
+
+type HandleSearchSubmit = (e?: React.MouseEvent<HTMLDivElement>) => void;
+type HandleSelectSubmit = (i: number) => (e: React.MouseEvent) => void;
+type HandleItemSubmit = (index: number) => void;
+
+export interface ISearchProps {
+    /** Current selected value */
```

Comment:
```suggestion
    /** Current value */
```
Выбирать не из чего, это не комбобокс.